README.md

```diff
@@ -68,7 +68,7 @@ As the start script runs, it will launch the Consul web UI and the Couchbase web
 
 ```
 Touchbase cluster is launched!
-Try scaling it up by running: ./start scale
+Try scaling it up by running: ./start.sh scale
```

Comment:
The `-f` flag can be left off and it's not supposed to show here if the user didn't use it in the initial `./start.sh` invocation.